server/src/main/java/org/apache/iotdb/db/qp/logical/Operator.java

```diff
@@ -77,7 +77,8 @@ public String toString() {
     TTL, DELETE_STORAGE_GROUP, LOAD_CONFIGURATION, SHOW, LOAD_FILES, REMOVE_FILE, MOVE_FILE, LAST, GROUP_BY_FILL,
     ALTER_TIMESERIES, FLUSH, MERGE, FULL_MERGE, CLEAR_CACHE,
     SHOW_MERGE_STATUS, CREATE_SCHEMA_SNAPSHOT, TRACING, DELETE_PARTITION,
-    CREATE_MULTI_TIMESERIES
-    , CREATE_INDEX, DROP_INDEX, QUERY_INDEX,
+    CREATE_MULTI_TIMESERIES, CREATE_INDEX, DROP_INDEX, QUERY_INDEX,
+    CHANGE_TAG_OFFSET, CHANGE_ALIAS, MNODE,
+    MEASUREMENT_MNODE, STORAGE_GROUP_MNODE;
```

Comment:
This could be removed, the operator is only generated in SQL parser.

Reply:
I use them to unify the wal and mlog, for deserializing the log.